extensions/gdx-setup/res/com/badlogic/gdx/setup/resources/html/build.gradle

```diff
@@ -15,30 +15,39 @@ gwt {
 }
 
 import org.wisepersist.gradle.plugins.gwt.GwtSuperDev
+import org.akhikhl.gretty.AppBeforeIntegrationTestTask
 
-def HttpFileServer server = null
-def httpFilePort = 8080
+gretty.httpPort = 8080
+gretty.resourceBase = project.buildDir.path + "/gwt/draftOut"
+gretty.contextPath = "/"
+gretty.portPropertiesFileName = "TEMP_PORTS.properties"
 
 task startHttpServer () {
     dependsOn draftCompileGwt
 
-    String output = project.buildDir.path + "/gwt/draftOut"
-
-    doLast {
+    doFirst {
         copy {
             from "webapp"
-            into output
+            into gretty.resourceBase
         }
 
         copy {
             from "war"
-            into output
+            into gretty.resourceBase
         }
+    }
+}
 
-        server = new SimpleHttpFileServerFactory().start(new File(output), httpFilePort)
+task beforeRun(type: AppBeforeIntegrationTestTask, dependsOn: startHttpServer) {
+    // The next line allows ports to be reused instead of
+    // needing a process to be manually terminated.
+    file("build/TEMP_PORTS.properties").delete()
+    // Somewhat of a hack; uses Gretty's support for wrapping a task in

```

Comment:
I should mention that the "hack code" was kinda the last resort I could come up with, after trying various ways around the very-brittle SimpleHttpFileServer code. The `integrationTestTask` isn't used as intended, but it does allow running two servers at once, a necessity for superDev because it has two different ports in use doing different things. Most of this change happened in May in gdx-liftoff, and as far as I can tell no one has had issues with superDev, including more than a few people who used liftoff in the latest jam.